src/validation/logic.rs

```diff
@@ -26,12 +26,13 @@ use amplify::confinement::{Confined, SmallBlob};
 use amplify::Wrapper;
 use strict_types::SemId;
 
+use crate::contract::operations::{Extension, Transition};
```

Comment:
I mean here it should be just
```suggestion
use crate::{Extension, Transition};
```